jena-arq/src/main/java/org/apache/jena/http/sys/ExecUpdateHTTPBuilder.java

```diff
@@ -275,25 +276,25 @@ public Y context(Context context) {
         if ( context == null )
             return thisBuilder();
         ensureContext();
-        this.context.setAll(context);
+        this.contextAcc.context(context);
         return thisBuilder();
     }
 
     public Y set(Symbol symbol, Object value) {
         ensureContext();
-        this.context.set(symbol, value);
+        this.contextAcc.set(symbol, value);
         return thisBuilder();
     }
 
     public Y set(Symbol symbol, boolean value) {
         ensureContext();
-        this.context.set(symbol, value);
+        this.contextAcc.set(symbol, value);
         return thisBuilder();
     }
 
     private void ensureContext() {
```

Comment:
The empty ensureContext method is akin to [ExecHTTPBuilder](https://github.com/apache/jena/blob/c7eaf83cf328496cb4f8a24b42a483199ac216c0/jena-arq/src/main/java/org/apache/jena/http/sys/ExecHTTPBuilder.java#L304). It could be removed there and in this file (`ExecUpdateHTTPBuilder).